_layouts/product-base.html.slim

```diff
@@ -7,42 +7,43 @@ secondary_section: supported-products
 /If you are aren't sure which product layout to use, you probably want 'product', not this one!
 - page.title = "#{page.product.abbreviated_name} - #{page.type}"
 
-.row.content
-  .large-24
-    ul.breadcrumbs itemscope="" itemtype="http://schema.org/BreadcrumbList"
-      li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem"
-        a itemprop="item" href="#{site.base_url}/products/" Products
-        meta itemprop="position" content="1"
-      li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem"
-        a href="#{site.base_url}/products/#{page.product.id}/overview" #{page.product.abbreviated_name}
-        meta itemprop="position" content="2"
-.row
-  .large-24.product-header
-    h2 #{page.product.name}
-    h3#product #{page.product.description}
+section(class="product-header #{page.product.id} #{page.product.product_category}")
+  .row
+    .large-24
+      ul.breadcrumbs itemscope="" itemtype="http://schema.org/BreadcrumbList"
+        li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem"
+          a itemprop="item" href="#{site.base_url}/products/" Products
+          meta itemprop="position" content="1"
+        li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem"
+          a href="#{site.base_url}/products/#{page.product.id}/overview" #{page.product.abbreviated_name}
+          meta itemprop="position" content="2"
+    .large-24
+      h2 #{page.product.name}
+      h3#product #{page.product.description}
 
-.row#hero-try-or-buy
-  .large-24.columns.hello-world
-    .small-24.large-6.columns
-      h2 Hello World!
-      p BUILD SOMETHING TODAY
-    .small-24.large-6.columns
-      p Let's walk through everything you need to build your first application.
-    .small-24.large-6.columns
-      a.button(href="../get-started/") Get Started
-    .small-24.large-6.columns.outside-links
-      a.large-24.medium-12.columns(href="http://www.redhat.com/en/about/contact/sales")
-        strong Buy It
-        p
-          | Contact Sales
-          i.fa.fa-caret-right
-      - if page.product.has_community_page
-        a.large-24.medium-12.columns(href="../community/")
-          strong Make it better
+section(class="product-graphics #{page.product.product_category}")
+  #hero-try-or-buy(class="row #{page.product.product_category}")
+    .large-24.columns.hello-world
+      .small-24.large-6.columns
+        h2 Hello World!
+        p BUILD SOMETHING TODAY
```

Comment:
This should be `Build Something Today` and then use CSS `text-transform:uppercase;` to make caps. It's a small thing, but helps with accessibility and readability